examples/embed/spectrogram/templates/spectrogram.html

```diff
@@ -5,13 +5,64 @@
     <meta http-equiv='content-type' content='text/html; charset=utf-8' />
 
     <title>Bokeh Spectrogram Demo</title>
+    
+    {% for raw in bokeh.css_raw %}
+        <style>{{ raw|safe }}</style>
+    {% endfor %}
+    <link href="static/css/styles.css" type="text/css" rel="stylesheet" />
 
-    {{ plot_resources|indent(4)|safe }}
-
-    {{ plot_script|indent(4)|safe }}
   </head>
   <body>
-    {{ plot_div|indent(4)|safe }}
-    <script type="text/javascript" src="static/js/spectrogram.js" defer></script>
+    {# start CONTENT BLOCK #}
+
+    {# bk-bs is bokeh's namespacing of normal bootstrap classes #}
+    <div class="bk-bs-container">
+        <h1>Bokeh Spectrogram demo</h1> 
+        <h3>with custom templating</h3>
+        
+        <div class="bk-bs-row">
+            <div class="bk-bs-col-sm-2">
+                <div id="freq-slider">
+                    {{ divs.freq_slider|safe }}
+                </div>
+                <div id="gain-slider">
+                    {{ divs.gain_slider|safe }}
+                </div>
+            </div>
+            <div class="bk-bs-col-sm-9">
+                <div id="spectrogram">
+                    {{ divs.spec|safe }}
+                </div>
+            </div>
+        </div>
+        <div class="bk-bs-row">
+            <div class="bk-bs-col-md-6">
+                <div id="signal">
+                {{ divs.signal|safe }}
+                </div>
+                <div id="spectrum">
+                {{ divs.spectrum|safe }}
+                </div>
+            </div>
+            <div class="bk-bs-col-md-5">
+                <div id="equalizer">
+                    {{ divs.equalizer|safe }}
+                </div>
+            </div>
+        </div>
+    </div>
+
+    {# end CONTENT BLOCK #}
+
+    {# start JS BLOCK #}
+
+        {% for raw in bokeh.js_raw %}
+            <script type="text/javascript">{{ raw|safe }}</script>
+        {% endfor %}
+
+        {{ script|indent(8)|safe }}
+        <script type="text/javascript" src="static/js/spectrogram.js" defer></script>
```

Comment:
The defer was already there I just copied and pasted. I'm almost certain it's not necessary as this is the last block in the page.